tests/Doctrine/Tests/Common/Annotations/AbstractReaderTest.php

```diff
@@ -408,6 +408,30 @@ public function testInvalidAnnotationButIgnored()
         self::assertEmpty($reader->getPropertyAnnotations($class->getProperty('foo')));
     }
 
+    public function testGloballyIgnoredAnnotationNotIgnored() : void
+    {
+        $reader = $this->getReader();
+        $class  = new \ReflectionClass(Fixtures\ClassDDC1660::class);
+
+        $active = true;
+
+        $testLoader = static function (string $className) use (&$active): bool {
+            if ($active && $className === 'since') {
+                throw new \InvalidArgumentException('Globally ignored annotation names should never be passed to an autoloader.');
+            }
+
+            return false;
+        };
+
+        \Doctrine\Common\Annotations\AnnotationRegistry::registerLoader($testLoader);
+
+        try {
+            self::assertNotNull($reader->getClassAnnotations($class));
```

Comment:
Unlike #235, in this case `getClassAnnotations()` returns non-empty array. I guess `testInvalidAnnotationButIgnored` is broken, but fixing it proper should be done in a different PR.